Artifacts/PgSql/Structure/Security/0050-Tables-MultipleAuthStrats.sql

```diff
@@ -0,0 +1,126 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+------------------------------ dbo.ResourceClaimActions -------------------------------------------------
+CREATE TABLE dbo.ResourceClaimActions(
+	ResourceClaimActionId SERIAL NOT NULL,
+	ActionId INT NOT NULL,
+	ResourceClaimId INT NOT NULL,
+	ValidationRuleSetName VARCHAR NULL,
+ CONSTRAINT ResourceClaimActionId_PK PRIMARY KEY
+(
+	ResourceClaimActionId 
+));
+
```

Comment:
We're missing a unique constraint/index, as documented in earlier comment.

Reply:
I created CREATE UNIQUE INDEX here